lib/fluent/root_agent.rb

```diff
@@ -64,26 +64,67 @@ def initialize(log:, system_config: SystemConfig.new)
     attr_reader :labels
 
     def configure(conf)
+      used_worker_ids = []
+      available_worker_ids = []
+      0.step(Fluent::Engine.system_config.workers - 1, 1).each do |id|
```

Comment:
Can't we use `available_worker_ids = (0..Fluent::Engine.system_config.workers - 1).to_a` like code?

Reply:
Oh.... We can use it. https://github.com/fluent/fluentd/pull/2292/commits/44f206a991a9b14971aaebdc0637bac9a1fb81e6